devicetypes/smartthings/motion-detector.src/motion-detector.groovy

```diff
@@ -17,6 +17,8 @@ metadata {
 		capability "Sensor"
 
 		fingerprint profileId: "0104", deviceId: "0402", inClusters: "0000,0001,0003,0009,0500"
+		//raw data: 01 C0C9 0001 01 03 0003 0005 0006 00
+		fingerprint profileId: "C0C9", deviceId: "0001", inClusters: "0003,0005,0006",  deviceJoinName: "Aurora Smart PIR Sensor"
```

Comment:
@tpmanley this skeeves me out. I think we should ask for new samples. It seems absurd that they wouldn't include manufacturer _or_ model.

Reply:
I didn't even look past that weird profile but yeah I agree

Reply:
ok, let's wait for upgraded samples / firmware update.